includes/wc-functions.php

```
@@ -309,7 +309,11 @@ function ( $value ) {
         update_post_meta( $post_id, '_backorders', $backorders );
 
         if ( $stock_status ) {
-            wc_update_product_stock_status( $post_id, $stock_status );
+            try {
+                wc_update_product_stock_status( $post_id, $stock_status );
+            } catch ( Exception $ex ) {
+                error_log( __FILE__ . ':' . __LINE__ . ' ' . $ex->getMessage() );
```

remove this error_log code from here, empty catch statement will do the tricks